_data/resources.yml

```diff
@@ -146,6 +146,9 @@
     - url: https://pressbooks.library.ryerson.ca/wafd/
       title: "Web Accessibility for Developers – Open Textbook"
       description: "by The Chang School, Ryerson University"
+    - url: https://silktide.com/resources/book
```

Comment:
Let's remove this for now. I don't feel comfortable promoting a book that isn't published yet without vetting its quality, especially if it's coming from a source that isn't primarily accessibility-focused.

Reply:
No problem. I'm waiting on that one for release maybe we can add it later.